final-react/src/utils/gameStats.jsx

```diff
@@ -1,12 +1,14 @@
 async function getTeamGameStats(week) {
   try {
+    const nflUrl = import.meta.env.VITE_NFL_URL + 'nfl/odds/json/TeamGameStats/2022REG/';
```

Comment:
this organization is much more clean and can prevent future issues with typos that can cause bugs. I don't see where the VITE file is but I'm guessing it is probably hidden. Might ask you in the future if I need the details.